src/features/hotspots/setup/HotspotSetupBluetoothError.tsx

```diff
@@ -19,68 +19,63 @@ const HotspotSetupBluetoothError = () => {
   const { params } = useRoute<Route>()
   const navigation = useNavigation<HotspotSetupNavigationProp>()
   return (
-    <Box flex={1}>
-      <Box flex={1}>
-        <Box marginBottom="l">
-          <Bluetooth />
+    <Box flex={1} justifyContent="center">
+      <Box marginBottom="l">
+        <Bluetooth />
+      </Box>
+      <Text
+        variant="h1"
+        maxFontSizeMultiplier={1}
+        numberOfLines={2}
+        adjustsFontSizeToFit
+        marginBottom="xxl"
+      >
+        {t('hotspot_setup.ble_error.title')} :(
+      </Text>
+
+      <Box flexDirection="row">
+        <Box
+          flex={3}
+          backgroundColor="purple200"
+          borderTopLeftRadius="m"
+          borderBottomLeftRadius="m"
+          padding="m"
+        >
+          <Text variant="body2Medium" marginBottom="xs">
+            {t('hotspot_setup.ble_error.enablePairing')}
+          </Text>
+          <Text variant="body2Light">
+            {t('hotspot_setup.ble_error.pairingInstructions')}
+          </Text>
         </Box>
-        <Text
-          variant="h1"
-          maxFontSizeMultiplier={1}
-          numberOfLines={2}
-          adjustsFontSizeToFit
-          marginBottom="xxl"
+        <Box
+          flex={1}
+          backgroundColor="purple300"
+          borderTopRightRadius="m"
+          borderBottomRightRadius="m"
+          alignItems="center"
+          justifyContent="center"
         >
-          {t('hotspot_setup.ble_error.title')} :(
-        </Text>
-
-        <Box flexDirection="row">
-          <Box
-            flex={3}
-            backgroundColor="purple200"
-            borderTopLeftRadius="m"
-            borderBottomLeftRadius="m"
-            padding="m"
-          >
-            <Text variant="body2Medium" marginBottom="xs">
-              {t('hotspot_setup.ble_error.enablePairing')}
-            </Text>
-            <Text variant="body2Light">
-              {t('hotspot_setup.ble_error.pairingInstructions')}
-            </Text>
-          </Box>
           <Box
-            flex={1}
-            backgroundColor="purple300"
-            borderTopRightRadius="m"
-            borderBottomRightRadius="m"
-            alignItems="center"
-            justifyContent="center"
-          >
-            <Box
-              width={28}
-              height={28}
-              borderRadius="round"
-              backgroundColor="purpleMain"
-              shadowColor="purpleMain"
-              shadowOpacity={1}
-              shadowOffset={{ width: 0, height: 0 }}
-              shadowRadius={8}
-            />
-          </Box>
+            width={28}
+            height={28}
+            borderRadius="round"
+            backgroundColor="purpleMain"
+            shadowColor="purpleMain"
+            shadowOpacity={1}
+            shadowOffset={{ width: 0, height: 0 }}
+            shadowRadius={8}
+          />
         </Box>
       </Box>
 
-      <Box justifyContent="flex-end">
```

Comment:
The buttons are no longer positioned at the bottom, which doesn't follow the mock up. 

For example the diagnostic screen while onboarding still had a bottom fixed button but centered text content.